src/StatusQ/Popups/statusModal/StatusImageWithTitle.qml

```diff
@@ -64,12 +66,35 @@ Row {
                 width: statusImageWithTitle.image.width
                 height: statusImageWithTitle.image.height
                 color: statusImageWithTitle.image.isIdenticon ?
-                    Theme.palette.statusRoundedImage.backgroundColor :
-                    "transparent"
+                           Theme.palette.statusRoundedImage.backgroundColor :
+                           "transparent"
                 border.width: statusImageWithTitle.image.isIdenticon ? 1 : 0
                 border.color: Theme.palette.directColor7
                 showLoadingIndicator: true
             }
+
+            StatusRoundButton {
+                type: StatusRoundButton.Type.Secondary
+
+                anchors.right: parent.right
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: -3
+
+                width: 15
+                height: 15
+
+                border.color: Theme.palette.indirectColor1
```

Comment:
I don't think we can do that and have to do no border here. It's hard to provide a border color with alpha channel that works across all backgrounds (avatar images):
<img width="130" alt="Screenshot 2021-10-13 at 14 04 00" src="https://user-images.githubusercontent.com/445106/137129443-9f5863f8-fe64-47d4-8335-295fe7bc1a3c.png">

So we either have to do a hard check against `Theme.palette.name` and give it a solid border color based on theme, or we remove it entirely

Reply:
okay